src/components/Widget.tsx

```diff
@@ -109,13 +81,6 @@ export interface TestableWidgetProps extends WidgetProps {
 }
 
 export function TestableWidget(props: PropsWithChildren<TestableWidgetProps>) {
-  const width = useMemo(() => {
-    if (props.width && props.width < 300) {
-      console.warn(`Widget width must be at least 300px (you set it to ${props.width}). Falling back to 300px.`)
-      return 300
-    }
-    return props.width ?? 360
-  }, [props.width])
   const locale = useMemo(() => {
```

Comment:
As a follow-up - in a separate PR, as this is unrelated - it would be nice to move this locale logic to the `I18nProvider` as well.

Reply:
https://github.com/Uniswap/widgets/pull/351